prow/config/config.go

```diff
@@ -180,6 +182,18 @@ type ExternalAgentLog struct {
 	URLTemplate *template.Template `json:"-"`
 }
 
+// Branding holds branding configuration for deck.
+type Branding struct {
```

Comment:
`omitempty` for all these.

Reply:
Done. I've actually moved the API into a separate PR: https://github.com/kubernetes/test-infra/pull/6272